WellCoffeeInventoryAPI/src/main/java/com/team_lightyear/WellCoffeeInventoryAPI/controllers/LoginController.java

```diff
@@ -1,4 +1,22 @@
 package com.team_lightyear.WellCoffeeInventoryAPI.controllers;
 
+import com.team_lightyear.WellCoffeeInventoryAPI.models.Account;
+import com.team_lightyear.WellCoffeeInventoryAPI.repositories.AccountRepository;
+import jakarta.validation.Valid;
+import jakarta.validation.constraints.Email;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Controller;
+import org.springframework.ui.Model;
+import org.springframework.validation.Errors;
+import org.springframework.web.bind.annotation.*;
+
+/**
+ * Created by Dominique Gould
+ */
+
+@Controller
+@RequestMapping("/api/")
```

Comment:
Is /api/ the final endpoint for this controller?

Reply:
Will work on LoginController on a different branch